packages/calcite-components/src/components/tab-nav/tab-nav.tsx

```diff
@@ -406,4 +516,55 @@ export class TabNav {
       tabTitles[this.selectedTabId].focus();
     });
   }
+
+  private getOverflowIcons(): VNode | VNode[] {
+    const { messages } = this;
+    const tabNavWidth = this.el.offsetWidth;
+
+    const tabTitles = Array.from(this.el.querySelectorAll("calcite-tab-title"));
```

Comment:
Looks like tab titles are queried here and inside `findVisibleTabTitleIndex`. Is there a way the query can be stored and used in both?